pkg/network/kuryr.go

```diff
@@ -0,0 +1,112 @@
+package network
+
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	uns "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	operv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/cluster-network-operator/pkg/bootstrap"
+	"github.com/openshift/cluster-network-operator/pkg/render"
+)
+
+// renderKuryr returns manifests for Kuryr SDN.
+// This creates
+// - the ClusterNetwork object
+// - the kuryr namespace
+// - the kuryr-controller deployment
+// - the kuryr-daemon daemonset
+// and some other small things.
+func renderKuryr(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.BootstrapResult, manifestDir string) ([]*uns.Unstructured, error) {
+	c := conf.DefaultNetwork.KuryrConfig
+	b := bootstrapResult.Kuryr
+
+	objs := []*uns.Unstructured{}
+
+	// render the manifests on disk
+	data := render.MakeRenderData()
```

Comment:
That comment was apparently copied from openshift_sdn.go, where it is also wrong. Please remove it.